guides/common/modules/proc_upgrading-packages-on-a-host.adoc

```diff
@@ -0,0 +1,16 @@
+[id="upgrading-packages-on-a-host_{context}"]
+= Upgrading Packages on a Host
+
+Use this procedure to upgrade packages on a host using the {ProjectWebUI}.
+The packages are upgraded through Katello agent or remote execution, depending on your configuration.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Click the name of the host you want to modify.
+. Go to *Content* tab, then click the *Packages* tab.
```

Comment:
```suggestion
. Select the *Content* tab, then select the *Packages* tab.
```

Reply:
Actually, I would prefer going with what was discussed in https://github.com/theforeman/foreman-documentation/pull/2117#discussion_r1156048203
```suggestion
. On the *Content* tab, navigate to the *Packages* subtab.
```